ajax/libs/sscaffold-css/package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Auto-update config is missing, please see CONTRIBUTING.md & CONTRIBUTING-WIP.md for more info.